### DISCUSSION TITLE
[Feature]: estimate initial depletion

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/FIMS#796] [Developers]
### Is your feature request related to a problem? Please describe.

In the Opakapaka case study, @MOshima-PIFSC notes she had to remove the estimation of initial depletion to simplify her model to run in FIMS

### Describe the solution you would like.

ability to estimate initial depletion within FIMS

### Describe alternatives you have considered

Don't estimate initial depletion

**Comment by kellijohnson-NOAA** [comment]
I moved this original issue about estimating initial depletion to a discussion rather than an issue so we can have a documented discussion about it. @MOshima-PIFSC can you provide a link to some documentation about how this is/was done in your case study. Does this pertain the age structured model or a surplus production model? Also @nathanvaughan-NOAA I think I have heard you talk about things like this before so perhaps you have some opinions.

**Comment by MOshima-PIFSC** [comment]
I don't have any formal documentation about this however, in the surplus production model we used in the stock assessment report, the psi parameter (ratio of initial biomass to carrying capacity) was estimated to be around 0.75 for Opakapaka. And, based on talking with fishers, we know there was some fishing prior to the start of when we have catch records. So to account for this in our age-structured model, we estimated an initial F parameter.
